tests/Avalonia.Controls.UnitTests/TextBoxTests.cs

```diff
@@ -460,6 +460,24 @@ public void Setting_Bound_Text_To_Null_Works()
             }
         }
 
+        [Fact]
+        public void Text_Box_MaxLength_Work_Properly()
+        {
+            using (UnitTestApplication.Start(Services))
+            {
+                var target = new TextBox
+                {
+                    Template = CreateTemplate(),
+                    Text = "abc",
+                    MaxLength = 3,
+                };
+
+                RaiseKeyEvent(target, Key.D);
```

Comment:
@benthecarman this is missing a parameter, can you change this `RaiseKeyEvent(target, Key.D, KeyModifiers.None);`?